tests/integration/targets/azure_rm_loganalyticsworkspace/tasks/main.yml

```diff
@@ -79,8 +79,8 @@
     that:
       - output.changed
       - output.retention_in_days == 30
-      # - output.intelligence_packs | json_query('[?name == `Backup`].enabled') | first == true
-      # - output.intelligence_packs | json_query('[?name == `Containers`].enabled') | first == true
+      - output.intelligence_packs | selectattr('name', 'equalto', 'Containers') | map(attribute='enabled') | first == true
```

Comment:
```suggestion
      - output.intelligence_packs | selectattr('name', 'equalto', 'Backup') | map(attribute='enabled') | first == true
```

Reply:
Sorry, there was an input error. It has been corrected